### ISSUE TITLE
No network connection ?

### ISSUE THREAD

**Issue by reddy9698** [botletics/SIM7000-LTE-Shield#290]
Hi, I've been using the module for quite some days now and it was working all fine.
Suddenly since yesterday the module doesn't seem to get a network connection.
I use the same SIM on my phone and it works fine but on the module the status LED keeps blinking every 1s.
The same SIM also works fine on other boards.
I've even tried using other SIM cards but to no avail.
Even tried changing the antenna and still the same.

Is it a problem with the board? Please Help

I even tried the following AT commands :
AT+CPIN?
+CPIN : READY

AT+CSQ
+CSQ : 99,99

AT+CGREG?
+CGREG : 0,2

AT+COPS?
+COPS: 0

**Comment by botletics** [comment]
So you mean the SIM card also works on other SIM7000 boards? It's likely a config issue. Please see the commands mentioned on [this wiki page] (https://github.com/botletics/SIM7000-LTE-Shield/wiki/Connecting-to-Network)

**Comment by reddy9698** [comment]
YES, the same sim card works on other boards, but not on my board.
I tried the link that you provided. 
AT command gives the following response : 

AT+CREG?
+CREG=0,0

AT+CREG=?
+CREG=(0-2)

**Comment by botletics** [comment]
Are you using a Botletics board?

**Comment by reddy9698** [comment]
Yes

**Comment by botletics** [comment]
Make sure you have a LiPo battery connected to it with proper polarity (red wire on left side of the connector when you plug the battery into the shield)

**Comment by reddy9698** [comment]
Yes, That's all the same. As I mentioned earlier, I have been working on it for quite some time now. It was working fine yesterday and suddenly the network dropped and I haven't gotten any connection since then.

**Comment by botletics** [comment]
Hmmm that's strange. You tried rebooting?

**Comment by reddy9698** [comment]
I tried rebooting, tried upgrading the firmware, tried many things but nothing seems to work.

**Comment by botletics** [comment]
Hmmm yea I'm not sure, sorry 😬. Can it still read the SIM card CCID?

**Comment by botletics** [comment]
Also, to get a baseline test, make sure to load the LTE_Demo sketch into the Arduino and test it, not whatever code you had on it before. Or unplug it from the Arduino altogether and test via USB.

**Comment by reddy9698** [comment]
No It returns an error for AT+CCID.
Yes I've tried both those methods.

**Comment by botletics** [comment]
Hmm, maybe somehow the SIM card circuitry is fried. Sounds unusual, but not impossible I guess... 🤔

**Comment by reddy9698** [comment]
Is there any way to check that?

**Comment by thesoaringpotato** [comment]
Perhaps try issuing a hard reset, then launch the LTE_Demo sketch and specifically copy and paste the results of each of the following commands:

[C] Read the SIM CCID
[i] Read signal strength (RSSI)
[n] Get network status
[1] Get network connection info

**Comment by reddy9698** [comment]
Tried that just now, Nothing's changed

SIM CCID : 

AT+CCID
ERROR

Signal Strength : 
AT+CSQ
+CSQ: 99,99

There's no further network status or connection info.
